jaggr-core/src/main/java/com/ibm/jaggr/core/impl/config/ConfigImpl.java

```diff
@@ -349,6 +349,7 @@ public int getExpires() {
 	/* (non-Javadoc)
 	 * @see com.ibm.jaggr.service.config.IConfig#getDeps()
 	 */
+	@Deprecated

```

Comment:
The javadoc for the interface does.  But the compiler warns if you override a deprecated interface without the annotation.